linera-core/src/client.rs

```diff
@@ -113,35 +119,33 @@ impl<ValidatorNodeProvider: Clone> Client<ValidatorNodeProvider> {
         self
     }
 
+    /// Returns the storage client used by this client's local node.
+    pub async fn storage_client(&self) -> S {
+        self.local_node.storage_client().await
+    }
+
     /// Creates a new `ChainClient`.
     #[allow(clippy::too_many_arguments)]
-    pub fn build<Storage>(
-        &self,
+    pub fn build(
```

Comment:
rename to `[make_]chain_client()` ?

Reply:
I was planning to do this a little later on, but I could do it here :)  I'm trying really hard to keep these PRs as minimal as possible while still passing tests.

Reply:
Addressed in https://github.com/linera-io/linera-protocol/pull/2133.